src/tutorial/scheduling/integer-cycling.rst

```diff
@@ -362,48 +362,32 @@ Recurrence Sections
            # Repeat every third cycle.
            P3 = baz
 
-.. digraph:: example
+.. image:: ../../img/reccurrance-sections.svg
```

Comment:
Sorry @wxtim - that filename reference is still spelt wrong :grin:  "...ence" !  (although now the actual filename is correct).

Reply:
Not to mention the double c.

Reply:
Oh yeah.  Geez, it's a tough word to get right :grin: